bin/remove.sh

```diff
@@ -1,26 +1,32 @@
-#!/bin/sh -f
+#!/bin/bash -f
 #
 # Used to remove indexes for a particular terminology/version.
 #
 help=0
 config=1
 ncflag=""
-stardog=0
+graph_db=0
 es=0
+
+l_graph_db_type=${GRAPH_DB_TYPE:-"stardog"}
+l_graph_db_home=""
```

Comment:
Here you're referencing a GRAPH_DB_TYPE variable but in setenv.sh_orig you are not setting GRAPH_DB_TYPE.    We probably should show that